extension/js/inspector/app/modules/Data/views/Layout.js

```diff
@@ -89,12 +93,24 @@ define([
       }
     },
 
+    showClassCollectionModels: function(opt) {
+      var filteredList = this.options.modelCollection.filter(function(model) {
+        return model.get('classId') === opt.classId
+      });
```

Comment:
I wonder if it would make sense to move this check to where `this.classCollection` is done. That way `ClassCollection` could be rendered directly.